conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+# Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost:8020/gobblin
+
+# Gobblin job configuration files
+export GOBBLIN_JOB_CONFIG_DIR=$GOBBLIN_WORK_DIR/jobs
```

Comment:
Sahil, thanks for your feedback!
1. I assume that the standalone mode should not write to hdfs.
2. I mentioned in one of my comments that probably GOBBLIN_WORK_DIR could be eliminated and the paths could be derived in the mr/standalone properties from ${fs.uri}.
The idea would be to have  global application-specific variables in gobblin-env (rather than in bashrc for instance) while the job-specific ones in the corresponding properties files.